bug-bounty-list/bug-bounty-list.json

```diff
@@ -129,6 +129,16 @@
     "hall_of_fame": false,
     "safe_harbor": "partial"
   },
+  {
+    "program_name": "Academy Mortgage",
+    "policy_url": "https://academymortgage.com/about-us/contact-us",
```

Comment:
@jswan106 https://academymortgage.com/about-us/contact-us doesn't list out the requirements for Partial Safe Harbor:

- Rewards: Whether compensation will be provided for (valid, unique) issues, as well as the form and magnitude of that compensation
- Official Communication Channels: An exhaustive list of the communication methods that are considered acceptable by the organization for receiving and communicating about any information associated with potential vulnerabilities
- Explicit permission to complete security research

Once these are met, we can add Academy Mortgage to the list, as well as marking Partial Safe Harbor.